worker-document-testing-unit/src/test/java/com/hpe/caf/worker/document/testing/CodeConfigurationSourceTest.java

```diff
@@ -52,7 +53,7 @@ public void testRetrievesAddedConfiguration() throws Exception
         assertThat(actualSource2, is(source2));
     }
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test
```

Comment:
If we're going to use `Assertions.assertThrows()` without perform additional checks on the exception instance then we should suppress the warning.

```suggestion
    @Test
    @SuppressWarnings("ThrowableResultIgnored")
```